app/src/main/res/values/strings.xml

```diff
@@ -617,7 +617,12 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_railway_crossing_barrier_none">It\'s not</string>
     <string name="action_deselect_all">Deselect all</string>
     <string name="quest_buildingType_retail_description">including buildings for restaurants, cafés, …</string>
-
+    <string name="quest_maxweight_title">"What is the weight limit here?"</string>
+    <string name="quest_maxweight_answer_noSign">There is no sign</string>
+    <string name="quest_maxweight_unusualInput_confirmation_description">This weight looks implausible. Are you sure that it is correct?</string>
+    <string name="quest_maxweight_answer_other_sign">There is some other weight limit sign…</string>
+    <string name="quest_maxweight_answer_sign_with_exceptions">There are exceptions…</string>
```

Comment:
Or "Depends on vehicle..."?

Reply:
Common ones are "Except inhabitants", "Except deliveries".

Maybe "It depends"?

Reply:
Oh, that is what you meant. Yeah, it depends is good, it covers all.

Reply:
Modified in cf4e9534